contrib/drivers/openstack/cinder/cinder.go

```diff
@@ -76,6 +79,36 @@ type CinderConfig struct {
 	Pool        map[string]PoolProperties `yaml:"pool,flow"`
 }
 
+//ListPoolOpts
+type ListPoolOpts struct {
+	// ID of the tenant to look up storage pools for.
+	TenantID string `q:"tenant_id"`
+	// Whether to list extended details.
+	Detail bool `q:"detail"`
+	//Volume_Type of the StoragePool
+	Volume_Type string `q:"volume_type"`
```

Comment:
It is possible to map multiple volume types to the same pool.  If you only have one volume type here, how do you capture other volume types also mapped to this pool if there are any?